### DISCUSSION TITLE
Livewire not working when manipulating the url

### DISCUSSION THREAD

**Discussion by ivinova** [livewire/livewire#3889] [1. Help]
I'm currently getting the following error in my console from Livewire ( v2.6.5 ):
`SyncBrowserHistory.js:196 Uncaught TypeError: Cannot create property 'livewire' on string 'data'`

Some parts of Livewire also seem to break, but with no clear reason why. I have the  @livewireStyles and @livewireScripts places correctly, and many parts of the application works perfectly fine.

UPDATE
I'm currently using JavaScript to manipulate the query part of the url. Eg `test.com?custom=3`. This package is all client side and via button clicks, it will remove all parameters from the url and re-add them. It uses the JavaScript ` window.history.pushState()`

The problem is that it breaks Livewire, and some parts break. How am I able to use Laravel wire trying to manipulate the url?

**Comment by ryatkins** [comment]
Can you use the Query String features of Livewire?
https://laravel-livewire.com/docs/2.x/query-string

Then you would just maninpulate a Livewire public property like `custom`
